apcd-cms/src/client/src/components/Admin/Extensions/AdminExtensions.tsx

```diff
@@ -33,18 +133,44 @@ export const AdminExtensions: React.FC = () => {
             <td>{row.ext_status}</td>
             <td>{formatDate(row.approved_expiration_date)}</td>
             <td className="modal-cell">
-              <select
-                id={`actionsDropdown_${row.ext_id}`}
-                onChange={`openAction('${row.ext_id}')`}
-              >
+                <select
+                  id={`actionsDropdown_${row.ext_id}`}
+                  defaultValue=""
+                  className="status-filter"
```

Comment:
```suggestion
```

Reply:
Removing these two classes will help the action select field stay within its container